src/library/scala/collection/Set.scala

```diff
@@ -28,6 +28,37 @@ trait Set[A]
 
   def canEqual(that: Any) = true
 
+  /**
+   * Equality of sets is implemented using the lookup method [[contains]]. This method returns `true` if
+   *   - the argument `o` is a `Set`,
+   *   - the two sets have the same [[size]], and
+   *   - for every `key` this set, `other.contains(key) == true`.
```

Comment:
Maybe `key` should be `element` for `Set` (then `k1, `k2` etc. should be `e1`, `e2`, etc.)